core/input/input.cpp

```diff
@@ -147,12 +147,29 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_accelerometer"), &Input::get_accelerometer);
 	ClassDB::bind_method(D_METHOD("get_magnetometer"), &Input::get_magnetometer);
 	ClassDB::bind_method(D_METHOD("get_gyroscope"), &Input::get_gyroscope);
+	ClassDB::bind_method(D_METHOD("is_joy_accelerometer_enabled", "device"), &Input::is_joy_accelerometer_enabled);
+	ClassDB::bind_method(D_METHOD("is_joy_gyroscope_enabled", "device"), &Input::is_joy_gyroscope_enabled);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("is_joy_sensors_enabled", "device"), &Input::is_joy_sensors_enabled);
```
See also https://github.com/godotengine/godot/pull/88590